src/main/java/uk/gov/justice/digital/hmpps/cmd/api/model/ShiftNotification.kt

```diff
@@ -21,10 +21,10 @@ data class ShiftNotification(
         var shiftModified: LocalDateTime,
 
         @Column
-        var taskStart: Int?,
+        var taskStart: Long?,
```

Comment:
isn't this a date type?

Reply:
Nothing comes to mind, I guess medium term we want to clean up this domain by returning a LocalTime from the stored procedures rather than seconds